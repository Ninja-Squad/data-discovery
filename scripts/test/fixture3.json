{
  "name": "123",
  "desc": "I'm a description!"
}