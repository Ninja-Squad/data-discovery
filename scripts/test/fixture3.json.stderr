[ERROR]: the input is expected to be an array in order to produce JSON lines bulk file.
